File: MT.Model/HTTTYT/MeasurementStandard.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region MeasurementStandard:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a MeasurementStandard:EntityBase.
    /// </summary>
    public class MeasurementStandard : EntityBase
    {
        [Key]
        public int MeasurementStandardID { get; set; }
        public string Code { get; set; }
        public string FullName { get; set; }
        public string ShortName { get; set; }
        public string Description { get; set; }



    }
    #endregion
}

File: BECore/app/model/MeasurementStandard.js
/// <summary>
/// Model EXTJS MeasurementStandard.
/// </summary>
Ext.define('QLDT.model.MeasurementStandard', {
    extend: 'Ext.data.Model',
    idProperty: 'MeasurementStandardID',
    idType: 'Guid',
    fields: [
        { name: "MeasurementStandardID", type: "int" },
        { name: "Code", type: "string" },
        { name: "FullName", type: "string" },
        { name: "ShortName", type: "string" },
        { name: "Description", type: "string" },
        { name: "CreatedDate", type: "date" },
        { name: "ModifiedDate", type: "date" },
        { name: "IPAddress", type: "string" },
        { name: "ModifiedBy", type: "string" },
        { name: "CreatedBy", type: "string" },
    ]
});

